wsproto/connection.py

```diff
@@ -186,36 +160,51 @@ def send_data(self, payload, final=True):
         self._outgoing += self._proto.send_data(payload, final)
 
     def close(self, code=CloseReason.NORMAL_CLOSURE, reason=None):
+        """
+        Initiate the close handshake by sending a CLOSE control message.
+
+        A clean teardown requires a CLOSE control messages from the other
+        endpoint before the underlying TCP connection can be closed, see
+        :class:`ConnectionClosed <wsproto.events.ConnectionClosed>`.
+        """
         self._outgoing += self._proto.close(code, reason)
         self._state = ConnectionState.CLOSING
 
     @property
     def closed(self):
         return self._state is ConnectionState.CLOSED
 
-    def bytes_to_send(self, amount=None):
+    def bytes_to_send(self, amt=None):
```

Comment:
Do we need to worry about this slightly-breaking change?

Reply:
@Lukasa if I revert this here, should I send a PR to h2 to rename it to `amount` as well?

Reply:
Sure, why not. =) Won't land until we decide to do a 4.0 though. ;)